---
- name: sahow the faile module
  hosts: all
  tasks:
  - name: do soemthing imposible
    lvol:
      lv: lvnothing
      vg: vgnothing
      size: 1G
    register: command_result
    ignore_errors: yes
  - name: show command_result error
    debug:
      var: command_result.err
  - name: print nice error message
    fail:
      msg: volume group doesn't exist
    when: "'not found' in command_result.err"      
